VAS.Core/Common/StateController.cs

```
@@ -131,7 +131,7 @@ public StateController ()
 				return ok;
 			} catch (Exception ex) {
 				Log.Exception (ex);
-				return false;
+				throw;
```

The problem is that this is swallowing the exceptions from the stateController itself.For example, if the first transition errors because it's not registered, you end up with an empty screen, no error message, and nothing to do.
I think it should throw, if the state wants to return false it can catch his own exception.